package.json

```diff
@@ -109,51 +109,22 @@
   },
   "fluidBuild": {
     "repoPackages": {
-      "client": [
-        {
-          "directory": "common"
-        },
-        {
-          "directory": "tools/generator-fluid"
-        }
+      "build": "common/build",
+      "common-def": "common/lib/common-definitions",
+      "common-utils": "common/lib/common-utils",
```

Comment:
I think the typing here of repoPackages: string | IFluidRepoPackage allows for conciseness where string automatically means the directory. However, I think we might want to keep the specification of "client" and "server" as specific parameters. Later on in the code, otherwise, you have to hardcode the "client" and "server" group checks, instead of just being able to check the .client and .server values

Reply:
I haven't done that in the PR, but in the next PR I'd like to make the code not specialize on "client" and "server"